### ISSUE TITLE
The page shown at http://rescience.github.io/ten-years/ is obsolete?

### ISSUE THREAD

**Issue by otizonaizit** [ReScience/ten-years#13]
Hi! It may be only me, but the page shown at http://rescience.github.io/ten-years/ is still the old one? The changes commited in March are still not online, I think...

**Comment by khinsen** [comment]
Right. @rougier, do your manual Web site updates cover the ten-years page as well?

**Comment by rougier** [comment]
It should have been updated. What were the March changes (I don't remember). In any case, we should at least mention the challenge is close and the workshop postponed.

**Comment by otizonaizit** [comment]
> It should have been updated. What were the March changes (I don't remember). In any case, we should at least mention the challenge is close and the workshop postponed.

Well, it was not updated. The changes in 852aa49fa52326b79da6c1a7b31f3167eab5881d are exactly adding that information ;-)

**Comment by otizonaizit** [comment]
Also, the author's guidelines introduced on December 11 with 2e905278ad8aea1b08f240b892ff7d5714b6b9aa are not linked from http://rescience.github.io/ten-years/ ?
But maybe I am not looking at the right page and the official challenge website is another one?

**Comment by rougier** [comment]
I see. There's the readme file on the ten years repository (https://github.com/ReScience/ten-years/blob/master/README.md) and a ten years page at the rescience website (https://github.com/ReScience/rescience.github.io/blob/sources/ten-years.md). Bad design (I'm responsible). Can you make a PR again the website ?

**Comment by otizonaizit** [comment]
> Can you make a PR again the website ?

done in https://github.com/ReScience/rescience.github.io/pull/80
